python/setup.py

```diff
@@ -199,7 +199,7 @@ def _supports_symlinks():
         extras_require={
             'ml': ['numpy>=1.7'],
             'mllib': ['numpy>=1.7'],
-            'sql': ['pandas']
+            'sql': ['pandas>=0.13.0']
```

Comment:
Actually, @BryanCutler, I just wonder if you are going to open a small follow up for Arrow and related minor doc changes?